packages/tests/src/_jest/helpers/mocks.ts

```diff
@@ -315,4 +322,4 @@ export const filterOutParticularities = (input: File) =>
     // Exclude webpack buildin modules, which are webpack internal dependencies.
     !input.filepath.includes('webpack4/buildin') &&
     // Exclude webpack's fake entry point.
-    !input.filepath.includes('fixtures/project/empty.js');
+    !input.filepath.includes('fixtures/empty.js');
```

Comment:
prefer `.endsWith` to `.includes`, to avoid something like `fixtures/empty.js/hello.js`

Reply:
fair.

Reply:
🤔 actually, given the context, I think we might also get `fixtures/empty.js?something` since it's within the bundling.
I've seen weird shit with the commonjs plugins for instance 😄
I prefer to filter out any mention of this path, which is also pretty much well under control.